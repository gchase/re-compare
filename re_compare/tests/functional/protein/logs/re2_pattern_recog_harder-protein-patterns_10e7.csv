regex,matchtype,matchnumber,time(ms),span
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),first_match,0,36.90385818481445,"[-1, -1]"
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),all_matches,0,36.90385818481445,"[-1, -1]"
DHKNLD.D,first_match,1,0.8089542388916016,"[613257, 613264]"
DHKNLD.D,consecutive,1,0.8089542388916016,"[613257, 613264]"
DHKNLD.D,consecutive,2,0.0059604644775390625,"[613397, 613404]"
DHKNLD.D,all_matches,2,0.8149147033691406,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,first_match,1,0.5910396575927734,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,1,0.5910396575927734,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,2,0.010967254638671875,"[99268, 99274]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,all_matches,2,0.6020069122314453,"[-1, -1]"
(GI)(G|R)P.Y..K..R,first_match,1,0.881195068359375,"[214952, 214963]"
(GI)(G|R)P.Y..K..R,consecutive,1,0.881195068359375,"[214952, 214963]"
(GI)(G|R)P.Y..K..R,consecutive,2,0.010967254638671875,"[215383, 215394]"
(GI)(G|R)P.Y..K..R,all_matches,2,0.8921623229980469,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),first_match,1,0.9400844573974609,"[140636, 140642]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),consecutive,1,0.9400844573974609,"[140636, 140642]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),consecutive,2,0.009059906005859375,"[140745, 140751]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),all_matches,2,0.9491443634033203,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),first_match,1,0.9860992431640625,"[201600, 201603]"
((N|S.TNH.Y)(F|W)N)(L|I),consecutive,1,0.9860992431640625,"[201600, 201603]"
((N|S.TNH.Y)(F|W)N)(L|I),consecutive,2,0.11181831359863281,"[226966, 226969]"
((N|S.TNH.Y)(F|W)N)(L|I),all_matches,2,1.0979175567626953,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,first_match,1,0.9109973907470703,"[157517, 157522]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,consecutive,1,0.9109973907470703,"[157517, 157522]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,consecutive,2,0.010251998901367188,"[157733, 157738]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,all_matches,2,0.9212493896484375,"[-1, -1]"
P..L...KWE.C,first_match,0,12.40396499633789,"[-1, -1]"
P..L...KWE.C,all_matches,0,12.40396499633789,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),first_match,0,45.77898979187012,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),all_matches,0,45.77898979187012,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,first_match,1,1.0230541229248047,"[811564, 811573]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,consecutive,1,1.0230541229248047,"[811564, 811573]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,consecutive,2,0.011920928955078125,"[812074, 812083]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,all_matches,2,1.0349750518798828,"[-1, -1]"
