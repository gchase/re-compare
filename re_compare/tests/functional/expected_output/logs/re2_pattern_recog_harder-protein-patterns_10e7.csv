regex,matchtype,matchnumber,time(ms),span
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),first_match,0,36.6208553314209,"[-1, -1]"
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),all_matches,0,36.6208553314209,"[-1, -1]"
DHKNLD.D,first_match,1,0.7369518280029297,"[613257, 613264]"
DHKNLD.D,consecutive,1,0.7369518280029297,"[613257, 613264]"
DHKNLD.D,consecutive,2,0.007152557373046875,"[613397, 613404]"
DHKNLD.D,all_matches,2,0.7441043853759766,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,first_match,1,0.698089599609375,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,1,0.698089599609375,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,2,0.012159347534179688,"[99268, 99274]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,all_matches,2,0.7102489471435547,"[-1, -1]"
(GI)(G|R)P.Y..K..R,first_match,1,0.6000995635986328,"[214952, 214963]"
(GI)(G|R)P.Y..K..R,consecutive,1,0.6000995635986328,"[214952, 214963]"
(GI)(G|R)P.Y..K..R,consecutive,2,0.009775161743164062,"[215383, 215394]"
(GI)(G|R)P.Y..K..R,all_matches,2,0.6098747253417969,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),first_match,1,1.8951892852783203,"[140636, 140642]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),consecutive,1,1.8951892852783203,"[140636, 140642]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),consecutive,2,0.024080276489257812,"[140745, 140751]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),all_matches,2,1.9192695617675781,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),first_match,1,1.5909671783447266,"[201600, 201603]"
((N|S.TNH.Y)(F|W)N)(L|I),consecutive,1,1.5909671783447266,"[201600, 201603]"
((N|S.TNH.Y)(F|W)N)(L|I),consecutive,2,0.1881122589111328,"[226966, 226969]"
((N|S.TNH.Y)(F|W)N)(L|I),all_matches,2,1.7790794372558594,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,first_match,1,1.3051033020019531,"[157517, 157522]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,consecutive,1,1.3051033020019531,"[157517, 157522]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,consecutive,2,0.02193450927734375,"[157733, 157738]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,all_matches,2,1.3270378112792969,"[-1, -1]"
P..L...KWE.C,first_match,0,16.67189598083496,"[-1, -1]"
P..L...KWE.C,all_matches,0,16.67189598083496,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),first_match,0,43.8230037689209,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),all_matches,0,43.8230037689209,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,first_match,1,0.9939670562744141,"[811564, 811573]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,consecutive,1,0.9939670562744141,"[811564, 811573]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,consecutive,2,0.010013580322265625,"[812074, 812083]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,all_matches,2,1.0039806365966797,"[-1, -1]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),first_match,1,0.7369518280029297,"[41351, 41359]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),consecutive,1,0.7369518280029297,"[41351, 41359]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),all_matches,1,47.4698543548584,"[-1, -1]"
(((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A))(L|I|V|M),first_match,0,10.207891464233398,"[-1, -1]"
(((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A))(L|I|V|M),all_matches,0,10.207891464233398,"[-1, -1]"
(A|G....GK)(S|T),first_match,1,0.2510547637939453,"[57, 58]"
(A|G....GK)(S|T),consecutive,1,0.2510547637939453,"[57, 58]"
(A|G....GK)(S|T),consecutive,2,0.023126602172851562,"[94, 95]"
(A|G....GK)(S|T),all_matches,2,0.2741813659667969,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)GT,first_match,1,2.2978782653808594,"[681321, 681326]"
((CP|.)(S|T)N)(I|L|V)GT,consecutive,1,2.2978782653808594,"[681321, 681326]"
((CP|.)(S|T)N)(I|L|V)GT,consecutive,2,0.21505355834960938,"[724976, 724982]"
((CP|.)(S|T)N)(I|L|V)GT,all_matches,2,2.5129318237304688,"[-1, -1]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,first_match,1,0.7340908050537109,"[413664, 413675]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,consecutive,1,0.7340908050537109,"[413664, 413675]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,consecutive,2,1.0647773742675781,"[1192336, 1192347]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,all_matches,2,1.798868179321289,"[-1, -1]"
